novuhq/website
# feat: pricing page data update

This pull request updates pricing page:

Steps to check:

- open [pricing preview](https://deploy-preview-138--novu-website.netlify.app/pricing/)
- make sure that everything looks and works as expected
